README.md

```diff
@@ -1,24 +1,66 @@
-# charmhub-lp-tools
+# Charmhub LP Tools
 
-Tools to configure and manage repositories and launchpad builders.
+Tools to configure and manage repositories and Launchpad builders.
 
 This repository provides the `charmhub-lp-tool` command that provides the
 ability to configure and manage the launchpad builders, repositories and
 branches in repositories.
 
 The commands are:
 
-* show -> display the current config.
-* list -> show a list of the charms configured in the supplied config.
-* diff -> show the current config and a diff to what is asked for.
-* config -> show the asked for config
-* sync -> sync the asked for config to the charm in the form of recipes.
+* `show`: The "show" command shows the current configuration for the charm
+  recipes as defined in launchpad.
+* `list`: List the charms defined in the configuration passed.
+* `diff`: Diff the declared config with the actual config in launchpad. This
+  shows the config and high-lights missing or extra configuration that is in
+  launchpad. Note that git repositories can have extra branches and these are
+  not seen in the diff. Missing branches that are in the config are
+  highlighted.
+* `sync`: Sync the config to launchpad. Effectively, this takes the diff and
+  applies it to the projects, creating or updating recipes as required.
+* `delete`: Delete a recipe from launchpad based on a track/risk. e.g. use
+  `--track` latest `--risk` edge to remove the recipe that pushes to the
+  latest/stable track. Note it does not remove the revision from the
+  charmhub. This is purely managing the recipes in launchpad.
+* `check-builds`: Check the state of the builds available at Launchpad
```

Comment:
Missing period/full stop at end of sentence.

Reply:
ack